js/scripts.js

```diff
@@ -1 +1,13 @@
 // target your button and assign an ask function that outputs a random image to your page
+
+
+// Click Event Listener with function(event)
+document.getElementById('askBtn').addEventListener("click", function(event) {
```

Comment:
since you're using the `onclick=""` attribute on the button, this event listener is not needed